### PR TITLE
Added separator margins for small devices

### LINKED ISSUE #217: Consider adding mobile behaviour for separator
smth like that

```
.mint-horizontal-separator--spaced {
    margin: $layoutDefaultPadding/3 0 2/3*$layoutDefaultPadding 0;
    @media (min-width: $breakpointTablet) {
      @extend .mint-horizontal-separator--spaced;
    }
  }
```

### PR DESCRIPTION
**Pull request by mr-mig**
Fixes #217 

image

### CODE REVIEW

**Comment by mr-mig** [comment]
Build process were changed to handle htmls inside components folders.

File: scripts/tasks/fingerprint-replace.js [hunk 1]
```diff
@@ -3,16 +3,24 @@ module.exports = function (gulp, plugins, consts) {
...
             .pipe(plugins.fingerprint(manifest, {
                 prefix: '../../',
                 regex: /(?:url\(["']?(.*?)['"]?\)|src=["'](.*?)['"]|src=([^\s\>]+)(?:\>|\s)|data=["'](.*?)['"]|href=["'](.*?)['"]|href=([^\s\>]+)(?:\>|\s))/g
             }))
             .pipe(gulp.dest(plugins.path.join(consts.VERSIONED_DIST, 'docs')));
+
+        return gulp.src(componentsHtmlPath)
+            .pipe(plugins.fingerprint(manifest, {
+                prefix: '../../../../',
+                regex: /(?:url\(["']?(.*?)['"]?\)|src=["'](.*?)['"]|src=([^\s\>]+)(?:\>|\s)|data=["'](.*?)['"]|href=["'](.*?)['"]|href=([^\s\>]+)(?:\>|\s))/g
```
Comment (kdzwinel) [inline scripts/tasks/fingerprint-replace.js]: Since we use the same regex twice, we should probably move it to a variable and reuse.

Comment (mr-mig) [inline scripts/tasks/fingerprint-replace.js]: Right, will do

File: src/components/separators/_horizontal-separator.scss [hunk 2]
```diff
@@ -14,7 +14,11 @@ $includeHtml: false !default;
     width: 100%;
 
     &--spaced {
-      margin: (rhythm(1) - $horizontalSeparatorHeight) 0 rhythm(1);
+      margin: rhythm(0.333) 0 rhythm(0.667);
```
Comment (aju) [inline src/components/separators/_horizontal-separator.scss]: This will not break a rhythm ?

**Comment by aju** [comment]
In general OK!

File: src/components/separators/_horizontal-separator.scss (hunk 2 above)
Comment (mr-mig) [inline src/components/separators/_horizontal-separator.scss]: The sum of margins will be `rhythm(1)`. 
But I believe I need to add `$horizontalSeparatorHeight` to equation. 
Thanks!

**Pull request merged**